File: Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/ItemManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IItemManager : IManagerBase<Item, string>
    {
	}
	
	partial class ItemManager : ManagerBase<Item, string>, IItemManager
    {
	}
}

File: trunk/Projects/Framework-Samples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/ItemManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IItemManager
	'	Inherits IManagerBase(Of Item, System.String)
	'End Interface

	Partial Class ItemManager
		Inherits ManagerBase(Of Item, System.String)
		Implements IItemManager
	End Class
End Namespace
